azure-cli/monitor/get_ApplicationInsights_ConnectionString.ps1

```diff
@@ -0,0 +1,21 @@
+function Get-ApplicationInsightsConnectionString {
+  param (
+    [Parameter(Mandatory = $true)]
+    [string]
+    $name,
+
+    [Parameter(Mandatory = $true)]
+    [string]
+    $resourceGroup
+  )
+
+  Write-Host "  Get connection string" -ForegroundColor DarkYellow
+  $connectionString = az monitor app-insights component show `
+    --app $name `
+    --resource-group $resourceGroup `
+    --query connectionString
```

Comment:
7.x is pscore! (also on Windows)

Reply:
This is the value on our app - looks fine to me:
InstrumentationKey=e4a3d9ca-2fd2-49f7-826c-31f71685f7f4;IngestionEndpoint=https://westeurope-5.in.applicationinsights.azure.com/

Reply:
It doesn't for me, I just tested both, with and without `--output tsv`, and as I expected, the one without returns a quoted result.
It may be true that the function you pass the variable to can deal with it being quoted, but it may confuse somebody down the road. 
Also in `New-ApplicationInsights.ps1` it has `--output tsv`, so this will just add more confusion

Reply:
Fixed